docs/standard/serialization/snippets/system-text-json-how-to-5-0/csharp/CustomConverterHandleNull.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace CustomConverterHandleNull
+{
+    public class Point
+    {
+        public int X { get; }
+        public int Y { get; }
+
+        public Dictionary<int, int> AdditionalValues;
+
+        [JsonConverter(typeof(DescriptionConverter))]
+        public string Description { get; set; }
```

Comment:
This is the only property needed for this example. Other properties can be removed to avoid distracting from the main scenario, particularly `AdditionalValues`.